hardhat.config.ts

```diff
@@ -1,14 +1,50 @@
-/**
- * @type import('hardhat/config').HardhatUserConfig
- */
-module.exports = {
+import '@nomiclabs/hardhat-waffle'
+import {existsSync, readFileSync} from 'fs'
+import {HardhatUserConfig} from 'hardhat/config'
+import {NetworkUserConfig} from 'hardhat/types'
+
+let infuraApiKey: string, mnemonic: string
+
+if (!existsSync('.infuraKey')) {
+  throw new Error("Put infura api key inside a .infuraKey file, don't paste it in the config.") // infura api key
+}
+
+if (!existsSync('.secret')) {
+  throw new Error("Put mnmenoic into a .secret file, don't paste it in the config.") // deployer wallet mnemonic
+}
+
+infuraApiKey = readFileSync('.infuraKey')
+  .toString()
+  .trim()
+
+mnemonic = readFileSync('.secret')
+  .toString()
+  .trim()
```

Comment:
why not use `const insfuraApiKey = ` and `const mnemonic = ` here, but use `let` first and assign later